machine_learning_hep/analysis/analyzer_jet.py

```diff
@@ -3144,12 +3400,16 @@ def jetsystematics(self):
             setup_legend(leg_sysvar)
             leg_sysvar.AddEntry(input_histograms_default[ibin2], "default", "P")
             setup_histogram(input_histograms_default[ibin2])
-            l_his_all = [his_var for l_cat in input_histograms_sys[ibin2] for his_var in l_cat] + [input_histograms_default[ibin2]]
+            l_his_all = []
+            for l_cat in input_histograms_sys[ibin2]:
+                for his_var in l_cat:
+                    if his_var.Integral()!=0:
```

Comment:
for all these conditions check that the indices of the arrays being filled are respected later